skfuzzy/control/controlsystem.py

```diff
@@ -329,28 +132,266 @@ def inputs(self, input_dict):
         for label, value in input_dict.items():
             self.input[label] = value
 
-    def view(self):
-        fig = ControlSystemVisualizer(self).view()
-        fig.show()
+    def compute(self):
+        """
+        Compute the fuzzy system.
+        """
+        # TODO: Tracking and caching
+
+        # Check if any fuzzy variables lack input values and fuzzyfy inputs
+        for antecedent in self.ctrl.antecedents:
+            if antecedent.input[self] is None:
+                raise ValueError("All antecedents must have input values!")
+            if antecedent.terms.values()[0].membership_value[self] is not None:
+                raise RuntimeError("Antecedent already has calculated "
+                "membership.  Are you trying to computer a simulation multiple "
+                "times?  Create multiple ControlSystemSimulation objects "
+                "instead.")
```

Comment:
IMO, iterative re-computation is a use case we should support without requiring new objects each time. 

Since we aren't worrying about caching at the moment, can we just overwrite these membership values and continue forward?